{
    "affected": [
        {
            "package": {
                "ecosystem": "photon:3.0",
                "name": "python-urllib3",
                "purl": "pkg:rpm/vmware/python-urllib3?distro=photon-3"
            },
            "ranges": {
                "events": [
                    {
                        "introduced": "0"
                    },
                    {
                        "fixed": "1.25.11-1.ph3"
                    }
                ],
                "type": "ECOSYSTEM"
            }
        },
        {
            "package": {
                "ecosystem": "photon:3.0",
                "name": "libxml2",
                "purl": "pkg:rpm/vmware/libxml2?distro=photon-3"
            },
            "ranges": {
                "events": [
                    {
                        "introduced": "0"
                    },
                    {
                        "fixed": "2.9.11-1.ph3"
                    }
                ],
                "type": "ECOSYSTEM"
            }
        }
    ],
    "id": "PHSA-2021-3.0-0246",
    "modified": "2023-05-01T05:24:55Z",
    "published": "2021-06-02T00:00:00Z",
    "references": [
        {
            "type": "ADVISORY",
            "url": "https://github.com/vmware/photon/wiki/Security-Update-3.0-246"
        }
    ],
    "related": [
        "CVE-2021-3537",
        "CVE-2021-3517",
        "CVE-2021-3518",
        "CVE-2019-11236",
        "CVE-2020-26137",
        "CVE-2019-11324"
    ]
}